sql/2021/caching/top_cdns.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  _TABLE_SUFFIX AS client,
+  cdn,
+  COUNT(0) AS n,
+  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct
```

Comment:
Check these `COUNT(0)s` are valid given the use of `UNNEST`

Reply:
Good catch. The total was coincidentally very close to the number of pages, but not correct. Fixed.